itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1316,7 +1307,7 @@ public NotificationEventResponse apply(@Nullable NotificationEventResponse event
     eventTypeValidator.assertInjectionsPerformed(true,false);
     InjectableBehaviourObjectStore.resetGetNextNotificationBehaviour(); // reset the behaviour
 
-    verifyRun("SELECT a from " + replDbName + ".ptned where (b=1) ORDER BY a", ptn_data, driverMirror);
+    verifyIfTableNotExist(replDbName , "ptned", metaStoreClientMirror);
```

Comment:
we are checking that insert event should not be generated which is great. but we should make sure that the data is replicated since add partition event will be generated and we should test atleast that the data is replicated, if not the stats, via alter_partition which fails.